google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -356,7 +356,7 @@ public void run() {
     }
   }
 
-  public SubscriberStats getStats() {
+  private SubscriberStats getStats() {
```

Comment:
We are tracking separately - let's not have unnecessary empty getters. Keeping the classes (but package private as you have them) seems fine.